eos-link-host-mali-driver

```diff
@@ -22,14 +22,75 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 EXTENSION_REF=org.freedesktop.Platform.GL.host/arm/1.6
-MALI_DIR=/usr/lib/mali400/arm-linux-gnueabihf
 
-DEST_DIR="/var/lib/flatpak/extension/$EXTENSION_REF"
-mkdir -p $(dirname "$DEST_DIR")
-ln -sf "$MALI_DIR" "$DEST_DIR"
+#
+# create_flatpak_gl_host_extension FLATPAK_DIR MALI_DRIVER
+#
+# Creates a flatpak GL host extension in $FLATPAK_DIR from $MALI_DRIVER
+#
+function create_flatpak_gl_host_extension {
+    FLATPAK_DIR=$1
+    MALI_DRIVER=$2
+
+    if test ! -d $FLATPAK_DIR; then
+        return;
+    fi
+
+    DEST_DIR=$FLATPAK_DIR/extension/$EXTENSION_REF
+
+    # Ensure destination directory
+    mkdir -p $DEST_DIR
+
+    # Create lib link for flatpak GL extension
+    ln -sr $DEST_DIR $DEST_DIR/lib
```

Comment:
This link is already exists in the host ostree -- it is created by the mali package.